### PR TITLE
plugins: simple WMTS plugin

### PR DESCRIPTION
**Pull request by alcalin**
Add link to a simple WMTS plugin for leaflet.

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ Ready-to-go basemaps, with little or no configuration at all.
         </td><td>
         	<a href="https://github.com/MIERUNE">Mierune</a>
         </td>
-    </tr>
+   	</tr>
```
Comment (johnd0e) [inline docs/plugins.md:226]: Please exclude these whitespace-only changes

File: docs/plugins.md [hunk 2]
```diff
@@ -214,7 +214,7 @@ Ready-to-go basemaps, with little or no configuration at all.
         </td><td>
         	<a href="https://github.com/SuperMap">SuperMap</a>
         </td>
-    </tr>
+   	</tr>
```
Comment (johnd0e) [inline docs/plugins.md:217]: Please exclude these whitespace-only changes

File: docs/plugins.md [hunk 3]
```diff
@@ -371,6 +371,14 @@ Plugins for loading basemaps or GIS raster layers in common (albeit non-default)
...
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/alcalin/L.TileLayer.WMTS">L.TileLayer.WMTS</a>
+		</td><td>A simple WMTS Tile Layer plugin for Leaflet.
+		</td><td>
+			<a href="https://github.com/alcalin">Alexandru Calin</a>
+		</td>
+	</tr>
```
Comment (johnd0e) [inline docs/plugins.md:381]: May be place it near leaflet.wms?

**Review by johnd0e — CHANGES_REQUESTED** [review]

File: docs/plugins.md (hunk 3 above)
Comment (alcalin) [reply docs/plugins.md]: Yes sir!

File: docs/plugins.md (hunk 1 above)
Comment (alcalin) [reply docs/plugins.md]: Done

File: docs/plugins.md (hunk 2 above)
Comment (alcalin) [reply docs/plugins.md]: Done

**Review by johnd0e — APPROVED** [review]

**Comment by MundoGIS** [comment]
If you are struggling with client-side lag due to heavy vector layers (GeoJSON) or slow WMS responses in your Leaflet projects, you should definitely check out Qtiles.

Qtiles allows you to generate standard XYZ tile sets (directory structure or MBTiles) directly from QGIS. This allows you to serve your map simply using L.tileLayer without needing complex server-side rendering.

Key benefits:

🚀 Unmatched Performance: We have tested these tiles with Leaflet, OpenLayers, Origo-map, and QGIS. The loading speed significantly outperforms middleware solutions like GeoWebCache, GeoServer, and MapProxy.

🎨 Custom Basemaps: Bake your complex QGIS styling directly into a fast, static base layer. No need to replicate complex styles in CSS/JS.

✅ Proven Stability: Since the tiles are pre-rendered, you remove the processing overhead from your server entirely.

Give it a try! https://github.com/nextgis/Qtiles

Let me know if you need any assistance setting this up.

BR/ Abel

**Pull request merged**